pkg/scheduler/algorithm/predicates/predicates.go

```diff
@@ -370,6 +377,8 @@ func getMaxVolumeFunc(filterName string) func(node *v1.Node) int {
 			return DefaultMaxGCEPDVolumes
 		case AzureDiskVolumeFilterType:
 			return DefaultMaxAzureDiskVolumes
+		case CinderVolumeFilterType:
+			return volumeutil.DefaultMaxCinderVolumes
```

Comment:
Unrelated: Why do we have `DefaultMaxCinderVolumes` coming from volumeutil, whereas rest(like DefaultMaxAzureDiskVolumes etc) are coming from the current file

Reply:
I think it is an oversight. All of these defaults should come from volumeutil. I will move them over.

Reply:
Or more specifically - `volumeutil.attach_limit.go` was introduced as common lib file that could be shared between plugin and scheduler. Most plugins use this file to derive keys and default limits. But scheduler already had those constants and they weren't removed and replaced when this new file was created.

Reply:
ok.